File: Assignments/Assignment_16/p8.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        int size=(row*2)-1,num=row;
        for(int i=0;i<row;i++){

                for(int j=1;j<=size;j++){

                        if((j<(i+1))||(j>(size-i))){

                                printf("\t");
                        }else{

                                printf("%d\t",(num-i));
                        }
                }printf("\n\n");
        }
}

File: Flutter/Assignment/Square pattern set-2/p8.dart
import "dart:io";
void main(){
	print("Enter the no. of rows:");
	int? row=int.parse(stdin.readLineSync()!);
	int y=1;
	for(int i=1;i<=row;i++){
		int x=1;
		for(int j=1;j<=y;j++){
			if(j%2==0){
				stdout.write("*\t");
			}else{
				stdout.write("${x}\t");
				x++;
			}
		}
		y+=2;
		print("");
	}
}
